src/components/board/Board.vue

```diff
@@ -23,6 +23,26 @@
 <template>
 	<div class="board-wrapper">
 		<Controls :board="board" />
+
+		<EmptyContent v-if="stacksByBoard.length === 0" icon="icon-pause">
+			{{ t('deck', 'No lists, add one') }}
```

Comment:
Let's do something more descriptive then here like:

**No lists available**
Create a new list to add cards to this board